src/compat/posix/stubs/utime.c

```diff
@@ -9,13 +9,15 @@
 #include <dirent.h>
 #include <errno.h>
 #include <kernel/printk.h>
-#include <stdio.h>
 #include <sys/types.h>
 #include <utime.h>
+#include <sys/stat.h>
+#include <fs/kfsop.h>
 
-int utime(const char *path,
-		const struct utimbuf *times) {
-	printk(">>> %s %s %p\n", __func__, path, times);
-	return SET_ERRNO(ENOSYS);
-}
+int utime(const char *path, const struct utimbuf *times) {
 
+	if (-1 == kutime(path, times)) {
```

Comment:
Why is there a real implementation in stubs folder. You should create a new module for your implementation.